p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/SocialSignIn/SocialSelection/SocialSignInViewModel.swift

```diff
@@ -53,10 +53,11 @@ class SocialSignInViewModel: NSObject, ViewModelType {
 
         input.onError.sink { [weak self] error in
             DispatchQueue.main.async {
-                self?.notificationService.showInAppNotification(.error(error))
+                self?.notificationService.showDefaultErrorNotification()
+//                self?.notificationService.showInAppNotification(.error(error))
```

Comment:
will it be relevant later?

Reply:
not sure, so decided to keep it just in case. maybe text will be changed or something